packages/build/src/steps/get.js

```diff
@@ -27,12 +31,40 @@ export const getDevSteps = function (command, steps) {
     coreStepName: 'dev.command',
     coreStepDescription: () => 'Run command for local development',
   }
-  const sortedSteps = sortSteps([...steps, devCommandStep], DEV_EVENTS)
+
+  const eventSteps = getEventSteps(eventHandlers)
+
+  const sortedSteps = sortSteps([...steps, eventSteps, devCommandStep], DEV_EVENTS)
   const events = getEvents(sortedSteps)
 
   return { steps: sortedSteps, events }
 }
 
+const getEventSteps = function (eventHandlers) {
+  return Object.entries(eventHandlers ?? {}).map(([event, eventHandler]) => {
+    let description = `Event handler for ${event}`
+    let handler = eventHandler
+
+    if (typeof eventHandler !== 'function') {
+      description = eventHandler.description
```

Comment:
I think this means that if you specify a `handler` property but no `description`, we'll end up with no description. Feels like we could use the default description in that case?

_Example_
```ts
await build({
  eventHandlers: {
    onPostBuild: {
       handler: () => {
         console.log('on post build');
       }
    },
  },
});
```

Reply:
Yeah! Good catch